backend/functions/lib/adapters/gateways/storage/local-storage-gateway.ts

```diff
@@ -12,15 +12,16 @@ const ROLE_MAPPING =
 
 export const OFFICE_MAPPING_PATH = '/officemapping.csv';
 const OFFICE_MAPPING =
-  'ad_group_name,idp_group_name,dxtr_court_id,dxtr_office_code\n' +
-  'USTP_CAMS_Region_2_Office_Manhattan,USTP CAMS Region 2 Office Manhattan,0208,1\n' +
-  'USTP_CAMS_Region_2_Office_Brooklyn,USTP CAMS Region 2 Office Brooklyn,0207,1\n' +
-  'USTP_CAMS_Region_2_Office_Central_Islip,USTP CAMS Region 2 Office Central Islip,0207,8\n' +
-  'USTP_CAMS_Region_2_Office_Albany,USTP CAMS Region 2 Office Albany,0206,1\n' +
-  'USTP_CAMS_Region_2_Office_Utica,USTP CAMS Region 2 Office Utica,0206,6\n' +
-  'USTP_CAMS_Region_2_Office_Buffalo,USTP CAMS Region 2 Office Buffalo,0209,1\n' +
-  'USTP_CAMS_Region_2_Office_Rochester,USTP CAMS Region 2 Office Rochester,0209,2\n' +
-  'USTP_CAMS_Region_2_Office_New_Haven,USTP CAMS Region 2 Office New Haven,0205,3\n';
+  'ad_group_name,idp_group_name,group_designator\n' +
+  'USTP_CAMS_Region_2_Office_Manhattan,USTP CAMS Region 2 Office Manhattan,NY\n' +
+  'USTP_CAMS_Region_2_Office_Brooklyn,USTP CAMS Region 2 Office Brooklyn,BR\n' +
+  'USTP_CAMS_Region_2_Office_Central_Islip,USTP CAMS Region 2 Office Central Islip,LI\n' +
+  'USTP_CAMS_Region_2_Office_Albany,USTP CAMS Region 2 Office Albany,AL\n' +
+  'USTP_CAMS_Region_2_Office_Utica,USTP CAMS Region 2 Office Utica,UT\n' +
+  'USTP_CAMS_Region_2_Office_Buffalo,USTP CAMS Region 2 Office Buffalo,BU\n' +
+  'USTP_CAMS_Region_2_Office_Rochester,USTP CAMS Region 2 Office Rochester,RO\n' +
+  'USTP_CAMS_Region_2_Office_New_Haven,USTP CAMS Region 2 Office New Haven,NH\n' +
+  'USTP_CAMS_Region_18_Office_Seattle,USTP CAMS Region 18 Office Seattle,SE|AK\n';
```

Comment:
Did we want to leave this example line in here?

Reply:
I'm not sure which line you're referring to.

Reply:
If it is accurate then I'm not opposed to leaving it.